src/visual/Paper.js

```diff
@@ -42,17 +42,30 @@ export default class Paper extends joint.dia.Paper {
         if (!cellViewT || !cellViewS || cellViewT.id === cellViewS.id) {
           return false;
         }
+
+        const isSuitablePortClass = (magnet) => {
+          const availableClasses = [
+            'port-body',
+            'port-body-empty',
+            'available-magnet',
+          ];
+
+          let result = true;
+          const portClasses = magnet.getAttribute('class').split(/\s+/);
+
+          _.forEach(portClasses, (portClass) => {
+            result = result && (availableClasses.indexOf(portClass) >= 0);
+          });
```

Comment:
Thank you so much for this advise. I promise I will use it in future in suitable situation not only at this code snippet (I hope it will be refactored by me in the next PR)